addons/io_scene_gltf2/io/com/gltf2_io_color_management.py

```diff
@@ -29,8 +30,53 @@ def color_linear_to_srgb(c):
     Convert from linear to sRGB color space.
 
     Source: Cycles addon implementation, node_color.h.
+    c may be a single color value or an array.
     """
-    if c < 0.0031308:
-        return 0.0 if c < 0.0 else c * 12.92
+    if type(c) in ([], np.ndarray):
+        colors = np.array(c, np.float32)
```

Comment:
If `c` is already a numpy array, this creates an unnecessary copy doesn't it?

Reply:
It just returns (a reference to) the original in that case.

Reply:
Pretty sure it returns a copy. You probably meant to pass `copy=False`.

Reply:
You're absolutely right. Will fix.